src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -3,103 +3,123 @@
 
 import os
 import unittest
+from functools import partial
 from textwrap import dedent
+from typing import Dict, List, Optional
 
 from pants.base.build_environment import get_buildroot
+from pants.option.option_value_container import OptionValueContainer
 from pants.option.options_bootstrapper import OptionsBootstrapper
 from pants.option.scope import ScopeInfo
 from pants.util.contextutil import temporary_dir, temporary_file, temporary_file_path
 
 
-class BootstrapOptionsTest(unittest.TestCase):
+class OptionsBootstrapperTest(unittest.TestCase):
 
-  def _do_test(self, expected_vals, config, env, args):
-    self._test_bootstrap_options(config, env, args,
-                                 pants_workdir=expected_vals[0],
-                                 pants_supportdir=expected_vals[1],
-                                 pants_distdir=expected_vals[2])
-
-  def _config_path(self, path):
+  def _config_path(self, path: Optional[str]) -> List[str]:
     if path is None:
       return ["--pants-config-files=[]"]
-    else:
-      return [f"--pants-config-files=['{path}']"]
-
-  def _test_bootstrap_options(self, config, env, args, **expected_entries):
+    return [f"--pants-config-files=['{path}']"]
+
+  def assert_bootstrap_options(
+    self,
+    *,
+    config: Optional[Dict[str, str]] = None,
+    env: Optional[Dict[str, str]] = None,
+    args: Optional[List[str]] = None,
```

Comment:
These default arguments are new and make the callsites much less verbose in some places.